src/Mod/BIM/ArchStairs.py

```diff
@@ -184,9 +186,12 @@ def setProperties(self,obj):
         if not "Landings" in pl:
             obj.addProperty("App::PropertyEnumeration","Landings","Structure",QT_TRANSLATE_NOOP("App::Property","The type of landings of these stairs"), locked=True)
             obj.Landings = ["None","At center","At each corner"]
-        if not "Winders" in pl:
-            obj.addProperty("App::PropertyEnumeration","Winders","Structure",QT_TRANSLATE_NOOP("App::Property","The type of winders in these stairs"), locked=True)
-            obj.Winders = ["None","All","Corners strict","Corners relaxed"]
+
+        # Not implemented yet, remarked out at the moment
+        #if not "Winders" in pl:
+        #    obj.addProperty("App::PropertyEnumeration","Winders","Structure",QT_TRANSLATE_NOOP("App::Property","The type of winders in these stairs"), locked=True)
+        #    obj.Winders = ["None","All","Corners strict","Corners relaxed"]
```

Comment:
This can be a problematic change. There are now older stairs that have this property. Newer stairs will not have it. Therefore future code will have to consider two possibilities.

Reply:
Thanks.  

I think when this is implemented at some point, the setProperties() method would be called by the onDocumentRestore() and this property would be added as it would required at that moment.  So it seems to be there would have no problem?  Am I missing something still?